### ISSUE TITLE
HTTP ERROR 500 After Plugin Activation (v.1.8.1)

### ISSUE THREAD

**Issue by mrlekman** [facebookincubator/facebook-for-woocommerce#306]
Hello, everything worked fine, but somehow now i get a 500 error if the Facebook For Woocommerce is activated. Do anyone have the same problem now? 
This is the error I get at logs:
AH01071: Got error 'PHP message: PHP Fatal error: Uncaught ArgumentCountError: Too few arguments to function WC_Facebookcommerce_EventsTracker::__construct(), 1 passed in /var/www/vhosts/domain.com/httpdocs/wp-content/plugins/facebook-for-woocommerce2/facebook-commerce.php on line 293 and exactly 2 expected in /var/www/vhosts/domain.com/httpdocs/wp-content/plugins/facebook-pixel/facebook-commerce-events-tracker.php:19\nStack trace:\n#0 /var/www/vhosts/domain.com/httpdocs/wp-content/plugins/facebook-for-woocommerce2/facebook-commerce.php(293): WC_Facebookcommerce_EventsTracker->__construct(Array)\n#1 /var/www/vhosts/domain.com/httpdocs/wp-content/plugins/woocommerce/includes/class-wc-integrations.php(39): WC_Facebookcommerce_Integration->__construct()\n#2 /var/www/vhosts/domain.com/httpdocs/wp-content/plugins/woocommerce/includes/class-woocommerce.php(464): WC_Integrations->__construct()\n#3 /var/www/vhosts/cante.gr/httpdocs/wp-includes/class-wp-hook.php(286): WooCommerce->init('')\n#4 /var/www/vhosts/domain.com/httpdocs/wp-includes/class-wp-...\n'

**Comment by mengyingdu** [comment]
Hi @mrlekman ,

Sorry to hear about this inconvenience for you. It seems you have installed 2 plugins, however, they should be used separated, and you'd better only keep 'facebook-for-woocommerce'. 
Steps to fix your problems:

In you dashboard, go to plugin tab:
1. deactivate 'facebook-pixel' and 'facebook-for-woocommerce', then click on delete for both of them
2. download plugin from [here](https://github.com/facebookincubator/facebook-for-woocommerce/releases/tag/v1.8.1)
3. still in plugin tab, 'add new plugin' --> 'upload' --> 'install' --> activate

Let me know if you need more help.

Thanks.

**Comment by mengyingdu** [comment]
Hi @mrlekman ,

Have you fixed your problems?

**Comment by dmitridr** [comment]
Hello. Two thoughts here
(1) `facebook-for-woocommerce2` doesn't sound like our plugin, but maybe a fork of our plugin?
(2) This specific error is due to PixelYourSite being installed with our plugin versions v1.7.9 (?) and below. So perhaps the fork of our plugin you are using mentioned in (1) needs to rebase on our changes. Alternatively, you may need to upgrade our plugin.

Hope that helps!
#235 has more discussion.

**Comment by mrlekman** [comment]
Hello, thank you all for your answers, I solved the problem. 
I made some tests, and found the problem, Facebook for Woocommerce works fine.
The facebook-for-woocommerce2 is the correct plugin, just for test I had to rename the folder. 
The problem was with the official Facebook Pixel plugin so I changed that plugin, and the problem solved. So hope no one had the same problems with his shop. 
Thanks again.
